marlowe-playground-client/src/Marlowe/Linter.purs

```diff
@@ -406,14 +406,24 @@ lint unreachablePaths contract =
   in
     CMS.execState (lintContract env contract) mempty
 
+lintParty :: Term Party -> CMS.State State Unit
+lintParty (Term (Address addr) pos) =
+  if validPaymentShelleyAddress addr then pure unit
+  else addWarning (InvalidAddress addr) pos
+
+lintParty (Term (Role role) _) =
```

Comment:
Role names are limited to 32 bytes (not 32 unicode characters) on the blockchain. Should the linter warn for names longer than that? Since addresses are specific to Cardano, it seems consistent to also check party names for consistency with Cardano.

Reply:
Thanks, good idea. Also policy ids, probably have to have the right length. I have added a ticket for it: [PLT-8827](https://input-output.atlassian.net/browse/PLT-8827)